apps/web/pages/_error.tsx

```diff
@@ -5,10 +5,23 @@ import Layout, { LayoutProps } from '../layouts/main'
 import I18n, { Locale } from '../i18n/I18n'
 import { withApollo } from '../graphql/withApollo'
 
+import {
+  Article,
+  ArticleCategory,
+  GetUrlQuery,
+  QueryGetUrlArgs,
+} from '@island.is/web/graphql/schema'
+import { GET_URL_QUERY } from '@island.is/web/screens/queries'
+import ApolloClient from 'apollo-client'
+import { NormalizedCacheObject } from 'apollo-cache-inmemory'
+import { NextPageContext } from 'next'
+import useRouteNames, { PathTypes } from '../i18n/useRouteNames'
+
 type ErrorPageProps = {
   statusCode: number
   locale: Locale
   layoutProps: LayoutProps
+  redirectProps: RedirectProps | null
```

Comment:
Aren't these props passed to ErrorPage.render?
ErrorPage doesn't need/accept redirectProps